### PR TITLE
Use Illuminate\Foundation\Testing\TestCase instead of Orchestra\Testbench\TestCase

### PR DESCRIPTION
**Pull request by crynobone**
Signed-off-by: Mior Muhammad Zaki <crynobone@gmail.com>

### CODE REVIEW

File: tests/Integration/Queue/CustomPayloadTest.php [hunk 1]
```diff
@@ -4,12 +4,15 @@
 
 use Illuminate\Contracts\Bus\QueueingDispatcher;
 use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Testing\TestCase;
 use Illuminate\Queue\Queue;
 use Illuminate\Support\ServiceProvider;
-use Orchestra\Testbench\TestCase;
+use Orchestra\Testbench\Concerns\CreatesApplication;
```
Comment (crynobone) [inline tests/Integration/Queue/CustomPayloadTest.php:10]: This should be testing `Illuminate\Foundation\Testing\TestCase` and not `Orchestra\Testbench\TestCase`.

I would recommend adding similar tests on `orchestra/testbench-core` to cover this issue.

Comment (deleugpn) [reply tests/Integration/Queue/CustomPayloadTest.php:10]: `getPackageProviders` won't work after this change, right? I'll adjust it to manually register it

Comment (crynobone) [reply tests/Integration/Queue/CustomPayloadTest.php:10]: @deleugpn `getPackageProviders` exists in `Orchestra\Testbench\Concerns\CreatesApplication`.

Comment (crynobone) [reply tests/Integration/Queue/CustomPayloadTest.php:10]: For simple tests such as this, we don't need `Orchestra\Testbench\Concern\Testing` where all the setUp, tearDown, and all other feature customs to Testbench exists.

**Pull request merged**